browser/profiles/brave_renderer_updater.cc

```diff
@@ -115,8 +141,9 @@ void BraveRendererUpdater::UpdateRenderer(
   auto default_solana_wallet = static_cast<brave_wallet::mojom::DefaultWallet>(
       brave_wallet_solana_provider_.GetValue());
   bool brave_use_native_solana_wallet =
-      (default_solana_wallet ==
-           brave_wallet::mojom::DefaultWallet::BraveWalletPreferExtension ||
+      ((default_solana_wallet ==
+            brave_wallet::mojom::DefaultWallet::BraveWalletPreferExtension &&
+        !should_ignore_brave_wallet) ||
```

Comment:
I think we need to check if Phantom is installed for Solana wallet instead of MetaMask.

Reply:
Actually let's remove this check for Solana wallet until we see different compact issue surfaced.

Reply:
Also Solana has another wrapper like wallet-stadard and wallet-adapter(deprecated) so wallets are very unlikely to  conflict with each others.